.travis/script_behat.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+set -ev
+
+vendor/bin/behat
```

Comment:
I'd add this as custom composer script, so it can easily be executed on the command line.